pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -199,6 +197,24 @@ type PipeSpec struct {
 	Key string `json:"key"`
 }
 
+// KudoOperatorSpec describes a how a KUDO operator is installed
+type KudoOperatorTaskSpec struct {
+	// either repo package name, local package folder or an URL to package tarball
+	// +optional
+	Package string `json:"package"`
+	// +optional
+	InstanceName string `json:"instanceName"`
+	// a specific app version in the official repo, defaults to the most recent
+	// +optional
+	AppVersion string `json:"appVersion"`
+	// a specific operator version in the official repo, defaults to the most recent one
+	// +optional
+	OperatorVersion string `json:"operatorVersion"`
+	// if specified, will be used to resolve the package name, otherwise the locally configured one will be used
+	// +optional
+	RepoURL string `json:"repoURL"`
```

Comment:
Wasn't there a discussion about leaving this one out for now? The Operator has to be either locally available, or in one of the configured repositories?

Reply:
You're right, we discussed repo-urls and decided to tackle them later. Should I remove it for now?

Reply:
Yes, I think that'd be good. I'm not sure if and when we want to support that in the future, and as it's optional it's easy to add back if we need to